app/services/applications/query.rb

```diff
@@ -78,7 +78,7 @@ def where_participant_ids_in(participant_ids)
     end
 
     def order_by
-      sort_order(sort:, model: Application, default: { created_at: :asc })
+      sort_order(sort:, model: Application, default: Arel.sql("COALESCE(applications.accepted_at, applications.created_at) ASC"))
```

Comment:
same here around this being in the participant query instead and this should remain the same